Botusanu Darius-Gabriel/lab 4, 3.9 THIS ONE PLEASE/main.c

```diff
@@ -0,0 +1,77 @@
+#include <stdio.h>
+#include <stdlib.h>
+#define SIZE 200
+
+/*I can not understand the "expected ) bevore [" on line 27 or 28*/
+int max(a,b)
```

Comment:
Well, your arguments don't have a type... 

what type is `a` and `b` here?  

If you didn't get things in the lab, why not crawl yourself through the support pdf and see what you can learn from there? 
Or better yet, ask me